include/zephyr/logging/log_frontend_stmesp_demux.h

```diff
@@ -121,11 +121,14 @@ struct log_frontend_stmesp_demux_trace_point {
 	/** Flag indicating if trace point includes data. */
 	uint64_t has_data: 1;
 
-	/** Timestamp. */
-	uint64_t timestamp: 58;
+	/** Timestamp. 54 bits at 40MHz is >14 years. */
+	uint64_t timestamp: 54;
```

Comment:
I guess bitfield operation, especially non-aligned to 2^x can slow down operations a bit. How often is this structure used?

Reply:
It's not used much. Data is written when decoding STPv2 stream (STM data) and then read to decode it to human readable format.